### PR TITLE
Add a runtests.sh #6

### LINKED ISSUE #6: Add a runtests.sh
Similar to runtests.bat but for Unix based users

### PR DESCRIPTION
**Pull request by ndt93**
Fixes #6 

- Fix undefined constants error
- Add runtests.sh script and update README accordingly

### CODE REVIEW

File: test/runtests.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+javac  ../src/seedu/addressbook/Addressbook.java -d ../bin
+
+# run the program, feed commands from input.txt file and redirect the output to the actual.txt
+java -classpath ../bin seedu.addressbook.AddressBook < input.txt > actual.txt
+
+# compare the output to the expected output
+diff actual.txt expected.txt
+if [ $? -eq 0 ]
+then
+    echo "diff: no differences encountered"
```
Comment (damithc) [inline test/runtests.sh]: `"Test result: PASSED"`?

File: src/seedu/addressbook/AddressBook.java [hunk 2]
```diff
@@ -382,7 +391,7 @@ private static String executeAddPerson(String commandArgs) {
 
         // checks if args are valid (decode result will not be present if the person is invalid)
         if (!decodeResult.isPresent()) {
-            return getMessageForInvalidCommandInput(COMMAND_WORD_ADD, getUsageInfoForAddCommand());
+            return getMessageForInvalidCommandInput(COMMAND_ADD_WORD, getUsageInfoForAddCommand());
```
Comment (damithc) [inline src/seedu/addressbook/AddressBook.java]: Unrelated change. Probably your branch has fallen behind.

**Comment by damithc** [comment]
Some minor changes requested.

File: src/seedu/addressbook/AddressBook.java (hunk 2 above)
Comment (damithc) [inline src/seedu/addressbook/AddressBook.java]: Or was it a bug?

Comment (ndt93) [inline src/seedu/addressbook/AddressBook.java]: Yup it's a bug. My branch is up to date.

**Comment by ndt93** [comment]
@damithc I've made the changes.

File: test/runtests.sh [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+# run the program, feed commands from input.txt file and redirect the output to the actual.txt
+java -classpath ../bin seedu.addressbook.AddressBook < input.txt > actual.txt
+
+# compare the output to the expected output
+diff actual.txt expected.txt
+if [ $? -eq 0 ]
+then
+    echo "Test result: PASSED"
+fi
```
Comment (damithc) [inline test/runtests.sh]: `else`? :-)

Comment (ndt93) [inline test/runtests.sh]: All rights. Added `else`
